src/audio/buffer.c

```diff
@@ -31,7 +31,6 @@ struct comp_buffer *buffer_alloc(uint32_t size, uint32_t caps, uint32_t flags, u
 				 bool is_shared)
 {
 	struct comp_buffer *buffer;
```

Comment:
@marcinszkudlinski  Style note: "sparse:" is not really good prefix. First, the patch prefix should specify the component (e.g. "audio: buffer: " in this patch). And second, "sparse" is a generic tool for static analysis. Tracking the cached attribute is just one of many things sparse looks for, so sparse in this context is very misleading. sure we understand now as we've talked about the sparse annotations this month a lot, but next year, us (or other people) maintaining the code will not make the connection.